contracts/interfaces/IWETH9.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 /// @title Interface for WETH9
+// curious what does `9` mean?
```

Comment:
not sure! thats the name of THE wrapped eth contract tho...
https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2#code

Reply:
probably versioning